build.mk

```diff
@@ -85,14 +85,14 @@ run-selftest:
 
```

Comment:
The default config needs `CONFIG_RISCV_SBI_V01=y` otherwise it won't compile here.

```
7.660 arch/riscv/kernel/reset.c: In function 'machine_restart':
7.660 arch/riscv/kernel/reset.c:28:12: error: 'SBI_EXT_0_1_SHUTDOWN' undeclared (first use in this function)
7.660    28 |  sbi_ecall(SBI_EXT_0_1_SHUTDOWN, 0, type, reason, 0, 0, 0, 0);
```